hack/update-bazel.sh

```diff
@@ -28,5 +28,7 @@ rm -f "${KUBE_ROOT}/pkg/generated/openapi/zz_generated.openapi.go"
 kube::util::go_install_from_commit github.com/kubernetes/repo-infra/kazel 9ba3a24eeffafa3a794b9e7312103424e7da26e9
 kube::util::go_install_from_commit github.com/bazelbuild/rules_go/go/tools/gazelle/gazelle 82483596ec203eb9c1849937636f4cbed83733eb
 
+[[ -f "${KUBE_ROOT}/vendor/BUILD" ]] || touch "${KUBE_ROOT}/vendor/BUILD"
```

Comment:
I wasn't sure whether `git` would care if the mtime updated, but I guess it doesn't.